src/sap/fhir/model/r4/FHIRModel.js

```diff
@@ -89,6 +90,7 @@ sap.ui.define([
 			this._buildGroupProperties(mParameters);
 			this.oDefaultQueryParameters = mParameters && mParameters.defaultQueryParameters && mParameters.defaultQueryParameters instanceof Object ? mParameters.defaultQueryParameters : {};
 			this.bSecureSearch = mParameters && mParameters.search && mParameters.search.secure ? mParameters.search.secure : false;
+			this.bPreProcessNextLink = mParameters && mParameters.preProcessNextLink != undefined ? mParameters.preProcessNextLink : true;
```

Comment:
Question: Do we see an additional `preProcessingMode` in future? For such scenarios I tend to use an enum instead of boolean. 

**Scenario**:
The property is called `preProcessingNextLinkMode` and the enum is called "LinkProcessingMode". Currently, the enum would have two properties "None" and "Default". Or any other names you prefer.

Reply:
As per discussion this is no longer applicable